react-toggle-switch-component/my-app/src/ToggleSwitch.js

```diff
@@ -0,0 +1,28 @@
+import { useState } from "react";
+import './ToggleSwitch.css';
+
+export default function ToggleSwitch() {
+ const [isOnOff, setIsOnOff] = useState(false)
+
+ function handleClick() {
+   setIsOnOff(!isOnOff);
+ }
+
+ let text = ''
+  if (isOnOff === true) {
+  text = "On 😁"
+  } else if (isOnOff === false) {text = "Off 😪"}
```

Comment:
In these if statements, you can just check `if (isOnOff)` instead of `if (isOnOff === true)`. Since `isOnOff` is a boolean, it can be used directly in the `if` statement without the comparison to `true` or `false`.

Also, in the else block, you already know that `isOnOff` is false (because it wasn't true, which you verified in line 12), so you can just use `else` without the extra `if (isOnOff == false)` statement.

Reply:
Oh, okay.  I'll take a look and play with that.  Still don't totally know the rules on what you can omit in else statements etc.  Thx!